### PR TITLE
Release 0.16.2

### PR DESCRIPTION
**Pull request by cournape**

### CODE REVIEW

**Comment by cournape** [comment]
@JCorson

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,10 @@
+2016-08-30    0.16.2
+--------------------
+
+Enhancements:
+
+  * add support for cp35 and cp36 in _guess_platform_abi (#233)
```
Comment (JCorson) [inline CHANGELOG]: Nit: I think this change only adds support for cp36 (cp35 was already supported).

**Comment by JCorson** [comment]
Other than the one minor comment LGTM.

File: CHANGELOG (hunk 1 above)
Comment (cournape) [inline CHANGELOG]: I think we can live with that :)

**Pull request merged**